manifest.json

```diff
@@ -3157,6 +3386,614 @@
+                            "sha256": "f604b91d63d2f3d14419d772a74697ba2ddcc0a846709e201dff55b8147f08ce"
+                        }
+                    ] 
+                },
+                "1.0.0": {
+                    "releaseUrl": "https://github.com/badhaloninja/MaterialTipAutoScoop/releases/tag/v1.0.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/badhaloninja/MaterialTipAutoScoop/releases/download/v1.0.0/MaterialTipAutoScoop.dll",
+                            "filename": "MaterialTipAutoScoop.dll",
+                            "sha256": "d75dd288d04340f4c22d65ac2ada51def260d5c258ffb8ba6ec402a16a3cdbbf"
+                        }
+                    ] 
+                }
+            }
+        },
+        "me.badhaloninja.ExtraTypeAliases": {
+            "name": "ExtraTypeAliases",
+            "description": "Adds more type aliases to the component attacher",
+            "category": "Technical Tweaks",
+            "sourceLocation": "https://github.com/badhaloninja/ExtraTypeAliases",
+            "authors": {
+                "badhaloninja": {
+                    "url": "https://github.com/badhaloninja"
+                }
+            },
+            "versions": {
+                "1.0.0": {
+                    "releaseUrl": "https://github.com/badhaloninja/ExtraTypeAliases/releases/tag/v1.0.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/badhaloninja/ExtraTypeAliases/releases/download/v1.0.0/ExtraTypeAliases.dll",
+                            "filename": "ExtraTypeAliases.dll",
+                            "sha256": "ead86d2c05d8366b97aa434064601685fe28c5524ef7c3edc3e4c0dac0192404"
+                        }
+                    ] 
+                }
+            }
+        },
+        "me.badhaloninja.ExitNeosAndShutdown": {
+            "name": "ExitNeosAndShutdown",
+            "description": "Adds a button to the exitscreen that shuts down your computer",
+            "category": "Memes",
+            "sourceLocation": "https://github.com/badhaloninja/ExitNeosAndShutdown",
+            "authors": {
+                "badhaloninja": {
+                    "url": "https://github.com/badhaloninja"
+                }
+            },
+            "versions": {
+                "1.0.0": {
+                    "releaseUrl": "https://github.com/badhaloninja/ExitNeosAndShutdown/releases/tag/v1.0.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/badhaloninja/ExitNeosAndShutdown/releases/download/v1.0.0/ExitNeosAndShutdown.dll",
+                            "filename": "ExitNeosAndShutdown.dll",
+                            "sha256": "478372db2e907a53029c531751e95bc002a99ac8ac3b3c753b664fc991c3d533"
+                        }
+                    ] 
+                }
+            }
+        },
```

Comment:
Uses hardcoded neosdb urls & seems windows specific, so missing broken on linux flag, or making a OS check and adding `shutdown now` when running on linux